LICENSES/gpl-v2.only.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Will Github get that we use different licenses and correctly mark our repo if we use a LICENSE *directory* like this? It has automatic recognition of a `LICENSE` file, but not a directory, right? What's the best course of action here?